tutorials/fundamentals/typeof-date.md

```diff
@@ -0,0 +1,28 @@
+JavaScript date's are considered objects. As a result,
+using `typeof` would result in it returing a string, `'object'`.
```

Comment:
Bit of an awkward sentence because it makes it seem like `typeof` returning a string is surprising. Let's do something more like "The [`typeof` operator](https://masteringjs.io/tutorials/fundamentals/typeof) returns 'object' for dates, so you can't use `typeof` to distinguish whether a value is a date."